### ISSUE TITLE
Missing Test Case - 399. Evaluate Division

### ISSUE THREAD

**Issue by ehgh** [LeetCode-Feedback/LeetCode-Feedback#6971]
There is a solution that passes all current test cases but fails the proposed test case.
Problem: Evaluate Division

#### Your LeetCode username

ehsangh

#### Category of the bug
- [ ] Question
- [ ] Solution
- [ ] Language
- [X] Missing Test Cases 

#### Description of the bug

There is a solution that passes all current test cases but fails the proposed test case.

#### Code you used for Submit/Run operation

```
// Evaluate Division
class Solution:
    def calcEquation(self, equations: List[List[str]], values: List[float], queries: List[List[str]]) -> List[float]:
        neighbordict = collections.defaultdict(list)
        pathvaluesdict = {}
        for (s, d), val in zip(equations, values):
            neighbordict[s].append(d)
            neighbordict[d].append(s)
            pathvaluesdict[(s, d)] = val 
            pathvaluesdict[(d, s)] = 1 / val
            pathvaluesdict[(s, s)] = 1
            pathvaluesdict[(d, d)] = 1

        def dfs(source, dest, visited):
            nonlocal pathvaluesdict, neighbordict
            if (source, dest) in pathvaluesdict:
                return
            for neighbor in neighbordict[source]:
                if not neighbor in visited:
                    visited.add(neighbor)
                    dfs(neighbor, dest, visited)
                    if pathvaluesdict[(neighbor, dest)] != -1: 
                        pathvaluesdict[(source, dest)] = pathvaluesdict[(source, neighbor)] * pathvaluesdict[(neighbor, dest)]
                        break
            if (source, dest) not in pathvaluesdict:
                pathvaluesdict[(source, dest)] = -1
                    
        out = []
        for source, dest in queries:
            dfs(source, dest, set([source]))
            out.append(pathvaluesdict[(source, dest)])
        return out
```

#### Language used for code

Python

#### Expected behavior

For test case:
[["a","b"],["b","c"],["c","d"]]
[2.0,3.0,4.0]
[["b","d"],["a","d"]]

Expected:
[12.00000,24.00000]
output:
[12.00000,-1.00000]

**Comment by LC-Yani** [comment]
Thank you for reaching out to us, the judger appears to be working correctly, please feel free to refer to the discuss section of the problem to see how other users' have optimized their codes!

**Comment by ehgh** [comment]
Thank you for your response. There must have been a misunderstanding of the issue I opened. The issue is **NOT** regarding the judger but the **current **test case set**** run to check the code. I **proposed to add a test case** that will complete the test case set (_WITHOUT changing the judge_) to prevent some incorrect solutions from passing all the test cases. I included such example code that passes ALL the current test cases but fails the proposed test case (if fails because the existing correct judger provides a different answer than the proposed code.)
